CS fixes

Nyholm opened the pull request.

samdark commented on src/Request.php:
```diff
@@ -16,11 +16,11 @@
-     * @param array $headers Request headers
-     * @param string|null|resource|StreamInterface $body Request body
-     * @param string $version Protocol version
+     * @param string                               $method  HTTP method
```
This won't be parse-able via IDEs. At least PhpStorm.

Nyholm replied:
Yeah, I saw that. I will figure out what php-cs-fixer rule that disables this.

samdark commented on src/Uri.php:
```diff
@@ -309,7 +309,7 @@ private function filterQueryAndFragment($str): string
         }
 
-        return \preg_replace_callback('/(?:[^' . self::CHAR_UNRESERVED . self::CHAR_SUB_DELIMS . '%:@\/\?]++|%(?![A-Fa-f0-9]{2}))/', [__CLASS__, 'rawurlencodeMatchZero'], $str);
+        return \preg_replace_callback('/(?:[^'.self::CHAR_UNRESERVED.self::CHAR_SUB_DELIMS.'%:@\/\?]++|%(?![A-Fa-f0-9]{2}))/', [__CLASS__, 'rawurlencodeMatchZero'], $str);
```
And this is against formatting PSR.

Nyholm replied:
These changes are automatic. I need to fine the correct php-cs-fixer rule to change this. 

I do not want to undo your changes.

Nyholm commented:
For future reference: If you run `php-cs-fixer fix -vvv` it will say **why** they are fixing some files. It makes it easier to find the correct rule in the docs.

Nyholm commented:
I have addressed the issues. 

Travis is red because it claims it is a BC break. 
I did not change the behavior of `MessageTrait::$stream`, it was the documentation that was wrong the in the first place.
